cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -35,21 +38,27 @@ def report_menus(group, appliance):
 
 def crud_files_reports():
     result = []
-    if not report_crud_dir.exists:
-        report_crud_dir.mkdir()
-    for file_name in report_crud_dir.listdir():
+    if not REPORT_CRUD_DIR.exists:
+        REPORT_CRUD_DIR.mkdir()
+    for file_name in REPORT_CRUD_DIR.listdir():
         if file_name.isfile() and file_name.basename.endswith(".yaml"):
             result.append(file_name.basename)
     return result
 
 
 @pytest.fixture(params=crud_files_reports())
 def custom_report_values(request):
-    with report_crud_dir.join(request.param).open(mode="r") as rep_yaml:
+    with REPORT_CRUD_DIR.join(request.param).open(mode="r") as rep_yaml:
         return yaml.safe_load(rep_yaml)
 
 
-@pytest.mark.tier(3)
+@pytest.fixture(scope="function")
+def get_custom_report(appliance, custom_report_values):
+    custom_report = appliance.collections.reports.create(**custom_report_values)
+    yield custom_report
+    custom_report.delete()
```

Comment:
Safer to call `delete_if_exists` here, which should be available on the entity as long as the exists method is implemented.